ports/pulsar-client-cpp/unofficial-pulsar-config.cmake

```
@@ -0,0 +1,58 @@
+if (NOT TARGET unofficial::pulsar::pulsar)
+    get_filename_component(VCPKG_IMPORT_PREFIX "${CMAKE_CURRENT_LIST_FILE}" PATH)
+    get_filename_component(VCPKG_IMPORT_PREFIX "${VCPKG_IMPORT_PREFIX}" PATH)
+    get_filename_component(VCPKG_IMPORT_PREFIX "${VCPKG_IMPORT_PREFIX}" PATH)
+
+    find_path(_pulsar_include_dir NAMES "pulsar/Client.h" PATH ${VCPKG_IMPORT_PREFIX})
```

```suggestion
    find_path(_pulsar_include_dir NAMES "pulsar/Client.h" PATH "${VCPKG_IMPORT_PREFIX}/include" REQUIRED)
```
BTW which style is officially to be used used:
~~~
include <pulsar/Client.h>
vs.
include <Client.h>
~~~

---

`#include <pulsar/Client.h>` is recommended.